controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -1218,6 +1219,10 @@ func (r *BareMetalHostReconciler) manageHostPower(prov provisioner.Provisioner,
 	}
 
 	if provResult.ErrorMessage != "" {
+		if !desiredPowerOnState && desiredRebootMode == metal3v1alpha1.RebootModeSoft &&
+			info.host.Status.ErrorType != metal3v1alpha1.PowerManagementError {
+			provResult.ErrorMessage = clarifySoftPoweroffFailure + provResult.ErrorMessage
```

Comment:
nit: if it's used just here a const could be avoided for an hardcoded Sprintf

Reply:
It is not only used here, but also in the unit test